sites/partners/src/components/listings/PaperListingDetails/sections/DetailUnits.tsx

```diff
@@ -60,10 +60,7 @@ const DetailUnits = ({ setUnitDrawer }: DetailUnitsProps) => {
     return t("t.none")
   }, [listing])
 
-  const enableHomeType =
-    profile?.jurisdictions
-      ?.find((j) => j.id === listing.jurisdictions.id)
-      ?.featureFlags?.find((flag) => flag.name === "homeType")?.active || false
+  const enableHomeType = doJurisdictionsHaveFeatureFlagOn("homeType", listing.jurisdictions.id)
```

Comment:
@YazeedLoonat I have added the util function for home type as mentioned https://github.com/housingbayarea/bloom/pull/830/files/93e3e7390be46afca3fb242d5ae09cd8b28fe90a#r1915735487